solutions/HodlCapsule.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity 0.4.24;
 
 contract Capsule {
 
-    //declare the following variables: owner_, amount_ and unlocktime_
+    //declared variables: owner_, amount_ and unlocktime_
```

Comment:
am thinking we don't actually need this comment if we are not going to have the students create the vars